app/models/manageiq/providers/azure/cloud_manager/metrics_capture.rb

```diff
@@ -209,8 +209,8 @@ def perf_collect_metrics(interval_name, start_time = nil, end_time = nil)
     counter_values_by_mor[ems_ref] = counter_values
 
     return counters_by_mor, counter_values_by_mor
-  rescue ::Azure::Armrest::BadRequestException # Probably means region is not supported
-    msg = "Problem collecting metrics for #{resource_description}. "\
+  rescue ::Azure::Armrest::BadRequestException => err # Probably means region is not supported
+    msg = "Problem collecting metrics for #{resource_description}: #{err}. "\
           "Region [#{provider_region}] may not be supported."
```

Comment:
When the region isn't supported for metrics is that obvious from the exception message?  Wondering if we can drop the extra "may not be supported"

Reply:
I can't check that, don't have credentials for such region (tried all Germany, according to https://github.com/ManageIQ/manageiq-providers-azure/pull/302). Dan, maybe you can find and show us, what the error do they return in such case? @djberg96

Reply:
I see @djberg96 said that `unfortunately the error message is not friendly.` so I'm good with this